packages/editor/src/components/block-settings-menu/index.js

```diff
@@ -38,7 +38,7 @@ const shortcuts = {
 	},
 	removeBlock: {
 		raw: rawShortcut.primaryAlt( 'backspace' ),
-		display: displayShortcut.primaryAlt( 'bksp' ),
+		display: displayShortcut.primaryAlt( 'Backspace' ),
```

Comment:
Might as well change this to the Mac one too, though I guess you'd need to add a new keyboard constant for it.

Reply:
Well I was going to, but then I discovered the `isMac` variable is not present in this component, and so instead of finding out how to make that happen, I had to move on to other work.

Given the fact that this shortcut will change in the very near future (it was already changed once and that change was reverted), I feel like it might not be worth it to change this when it needs to be something else? See discussion in https://github.com/WordPress/gutenberg/pull/9190.